README.md

```diff
@@ -1,84 +1,84 @@
-
-### Packages
-
-All packages maintained with this monorepo are listed below.
-
-| Package | Version |Description |
-| - | - | - |
-| [`ses`][ses-repo] | [![npm][ses-npm-svg]][ses-npm] | Secure ECMAScript. |
-| [`@agoric/harden`][harden-repo] | [![npm][harden-npm-svg]][harden-npm] | Build a defensible API surface around an object by freezing all reachable properties. |
-
-## Examples
-
-## Contributing
-
-### Branches
-
-#### `master`
-
-- Latest development code.
-
-#### `0.6-stable`
-
-- Original SES code which was build on the realm-shim and still releasable:
-
-```
-$ git checkout 0.6-stable
-$ cd packages/ses
-$ npm install
-$ npm run build
-
-src/index.js → dist/ses.esm.js, dist/ses.cjs.js...
-created dist/ses.esm.js, dist/ses.cjs.js in 220ms
-
-src/index.js → dist/ses.umd.js...
-created dist/ses.umd.js in 204ms
-```
+Endo is a JavaScript platform 👷‍♀️*under development*🏗 for secure communication
+among objects within one process and distributed between mutually suspicious
+machines.
+The foundation of Endo is [SES][], a tamper-proof JavaScript environment that
+allows safe execution of arbitrary programs in Compartments.
+
+⚠️Endo is not Done.⚠️
+
+Most JavaScript libraries built for Node.js, either in CommonJS or ECMAScript
+module format, are suitable for running in Endo without modification, since
+such programs rarely tamper with global scope or shared intrinsic objects.
+The exception is shims, which require special treatment and express consent to
+work with Endo.
+
+Endo protects program integrity both in-process and in distributed systems.
+SES protects local integrity, defending an application against [supply chain
+attacks][]: hacks that enter through upgrades to third-party dependencies.
+Endo does this by encouraging the [Principle of Least Authority][POLA].
+
+Since most JavaScript libraries receive powerful capabilities from global
+objects like `fetch` or modules like `net`, Endo uses [LavaMoat][] to
+automatically generate reviewable policies that determine what capabilities
+will be distributed to third party dependencies.
+
+For distributed systems, Endo stretches object oriented programming over
+networks using asynchronous message passing to remote objects with the
+[Handled Promise][] API and a [Capability Transfer Protocol][CapTP].
+
+Between remote objects and SES compartments, Endo makes distributed programs
+easy to program, and easy to reason about integrity.  CapTP frees the
+programmer from needing to create bespoke communication protocols over message
+ports or byte streams.
+
+[SES]: packages/ses
+[Handled Promise]: https://github.com/Agoric/agoric-sdk/tree/master/packages/eventual-send
+[CapTP]: https://github.com/Agoric/agoric-sdk/tree/master/packages/captp#agoriccaptp
+[LavaMoat]: https://github.com/LavaMoat/LavaMoat
+[POLA]: https://en.wikipedia.org/wiki/Principle_of_least_privilege
```

Comment:
I hate that this page doesn't distinguish POLA from POLP. But it is still probably the best one thing to cite. No change suggested. Just sayin.

Reply:
Time to put the wiki back in wikipedia.